src/nautilus-server/allowed_endpoints.yaml

```
@@ -1,3 +1,5 @@
 # External endpoints that the enclave is allowed to access. 
 endpoints:
   - api.weatherapi.com # replace with your own endpoints
+  - https://fullnode.mainnet.sui.io:443
+  - https://sui-mainnet.mystenlabs.com/graphql
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)